libraries/SITL/examples/Webots_Python/controllers/ardupilot_vehicle_controller/ardupilot_vehicle_controller.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+import time
+import argparse
+from webots_vehicle import WebotsArduVehicle
+
+#
+# General ardupilot vehicle controller for Webots 2023a
+#
```

Comment:
It would be nice if this (and the other Python files) was declared `AP_FLAKE8_CLEAN`, but not required.

Reply:
I shall look into that.